prefs.js

```diff
@@ -1,1232 +1,36 @@
-                'active',
-                Gio.SettingsBindFlags.DEFAULT
-            );
-            this._settings.bind(
-                'customize-alphas',
-                this._builder.get_object('min_alpha_scale'),
-                'sensitive',
-                Gio.SettingsBindFlags.DEFAULT
-            );
-            this._settings.bind(
-                'customize-alphas',
-                this._builder.get_object('max_alpha_scale'),
-                'sensitive',
-                Gio.SettingsBindFlags.DEFAULT
-            );
-
-            const minAlphaScale = this._builder.get_object('min_alpha_scale');
-            const maxAlphaScale = this._builder.get_object('max_alpha_scale');
-            minAlphaScale.set_value(
-                this._settings.get_double('min-alpha')
-            );
-            minAlphaScale.set_format_value_func((_, value) => {
-                return `${Math.round(value * 100)} %`;
-            });
-            maxAlphaScale.set_format_value_func((_, value) => {
-                return `${Math.round(value * 100)} %`;
-            });
-
-            maxAlphaScale.set_value(
-                this._settings.get_double('max-alpha')
-            );
-
-            dialog.connect('response', () => {
-                // remove the settings box so it doesn't get destroyed;
-                dialog.get_content_area().remove(box);
-                dialog.destroy();
-            });
-
-            dialog.present();
-        });
-
-
-        this._settings.bind('unity-backlit-items',
-            this._builder.get_object('unity_backlit_items_switch'),
-            'active', Gio.SettingsBindFlags.DEFAULT
-        );
-        this._settings.bind('apply-glossy-effect',
-            this._builder.get_object('apply_gloss_effect_checkbutton'),
-            'active', Gio.SettingsBindFlags.DEFAULT
-        );
-        this._settings.bind('unity-backlit-items',
-            this._builder.get_object('apply_gloss_effect_checkbutton'),
-            'sensitive',
-            Gio.SettingsBindFlags.DEFAULT
-        );
-
-        this._settings.bind('force-straight-corner',
-            this._builder.get_object('force_straight_corner_switch'),
-            'active', Gio.SettingsBindFlags.DEFAULT);
-
-        this._settings.bind('disable-overview-on-startup',
-            this._builder.get_object('show_overview_on_startup_switch'),
-            'active', Gio.SettingsBindFlags.INVERT_BOOLEAN);
-
-        // About Panel
+const Me = ExtensionUtils.getCurrentExtension();
 
-        if (Me)
-            this._builder.get_object('extension_version').set_label(Me.metadata.version.toString());
-        else
-            this._builder.get_object('extension_version').set_label('Unknown');
-    }
-});
+// import pages
+const { General } = Me.imports.preferences.general;
+const { Launchers } = Me.imports.preferences.launchers;
+const { Behavior } = Me.imports.preferences.behavior;
+const { Apparence } = Me.imports.preferences.apparence;
```

Comment:
```suggestion
const { Appearance } = Me.imports.preferences.appearance;
```

This would of course need the other occurrences and filename changed

Reply:
Thanks, I'll will change that.